packages/flutter/lib/src/widgets/visibility.dart

```diff
@@ -85,6 +85,28 @@ class Visibility extends StatelessWidget {
          'Cannot maintain interactivity if size is not maintained.',
        );
 
+  /// Control whether the given [child] is [visible].
+  ///
+  /// The [child] and [replacement] arguments must not be null.
+  ///
+  /// This is equivalent to the default [Visibility] constructor with all
+  /// "maintain" fields set to true. This constructor should be used in place of
+  /// an [Opacity] widget that only takes on values of `0.0` or `1.0`, as it
+  /// avoids extra compositing when fully opaque.
+  const Visibility.maintain({
```

Comment:
nit: Maybe `maintainAll` would be a little more explicit? Not feeling strongly, though.

Reply:
I would prefer the shorter name :)